soapui/src/main/java/com/eviware/soapui/impl/rest/actions/explorer/callback/EndpointExplorerCallback.java

```diff
@@ -207,6 +208,10 @@ public void exploreAPIClickBodyTab() {
         Analytics.trackAction(EXPLORE_API_CLICK_BODY_TAB);
     }
 
+    public void exploreAPIDontShowAgain(boolean newValue) {
+        SoapUI.getSettings().setBoolean(SHOW_ENDPOINT_EXPLORER_ON_START, !newValue);
+    }
```

Comment:
@AlexanderNikitin-Smartbear Does it make sense to make two-way binding to avoid adding such methods?